kolibri/plugins/management/assets/src/vue/class-enroll-page/index.vue

```diff
@@ -353,5 +354,63 @@
 
   .col-name, .col-username
     width: 45%
+    
+  .results-text
+    font-size: 0.9375rem
+    
+  .toolbar
+    margin-top: 30px
+    
+  .toolbar:after
+    content: ''
+    display: table
+    clear: both
+    
+  input[type='search']
```

Comment:
I see. This code is pretty rough - between the lack of class names, use of floats, and referencing media queries and breakpoint variables. On the other hand, there's a benefit to getting in in for consistency

two options:
 * easy: add some scary comments around it, saying that the code needs to be deleted asap
 * if the code is actually shared with another file, maybe move it to a shared .styl file?